gensim/models/word2vec.py

```diff
@@ -558,11 +807,13 @@ def _set_train_params(self, **kwargs):
     def train(self, sentences, total_examples=None, total_words=None,
               epochs=None, start_alpha=None, end_alpha=None, word_count=0,
               queue_factor=2, report_delay=1.0, compute_loss=False, callbacks=()):
-        """Update the model's neural weights from a sequence of sentences (can be a once-only generator stream).
-        For Word2Vec, each sentence must be a list of unicode strings. (Subclasses may accept other examples.)
+        """
+        Update the model's neural weights from a sequence of sentences. `sentences` may be a generator stream.
```

Comment:
please use `"""Update ...`, no need newline (here and everywhere when you have description of method).

Reply:
Is this a part of the NumPy standard?

Because if not, I'd prefer to have the first line indented the same way as all other lines. The inconsistency makes the docs harder to read (and the first line is usually the most important one).